bot.go

```diff
@@ -70,7 +70,7 @@ func (b *Bot) Run() {
 
 // Reply to a message and simulate typing through the realtime messaging API
 func (b *Bot) ReplyAndType(evt *slack.MessageEvent, text string) {
-	b.Type(evt.Channel, text)
+	b.Type(evt.Channel, text, 0)
```

Comment:
What if instead we set a max delay within `ReplyAndType` to be 1500 or 2000 ms? I'd rather not have to always pass that parameter and that would help for really long strings too.